.github/workflows/docs.yml

```diff
@@ -16,13 +17,29 @@ jobs:
     permissions:
       contents: write
       pull-requests: write
+    env:
+      BIN_DIR: ${{ github.workspace }}/bin
+      DOXYGEN_URL: 'https://github.com/doxygen/doxygen/releases/download/Release_1_10_0/doxygen-1.10.0.linux.bin.tar.gz'
+      DOXYGEN_VERSION: 1.10.0
```

Comment:
I think this is sus. I know `1.9.5` is minimum but I think we should keep the minimum and the version we download in sync if we're going with this route. It make more sense to me to download 1.9.5.

Reply:
Yes, makes sense! Either download the latest (so that CI can let people know when something starts to break) or pin at the minimum version.